src/main/java/org/ballerinalang/command/util/OSUtils.java

```diff
@@ -213,7 +214,29 @@ private static boolean isSolaris() {
     private static String getUserHome() {
         String userHome = System.getenv("HOME");
         if (userHome == null) {
-            userHome = System.getProperty("user.home");
+            if (OS.contains("nux")) {
+                ProcessBuilder processBuilder = new ProcessBuilder();
+                processBuilder.command("bash", "-c", "echo home/$USER");
+                StringBuilder output = new StringBuilder();
+                try {
+                    Process process = processBuilder.start();
+
+                    BufferedReader reader = new BufferedReader(
+                            new InputStreamReader(process.getInputStream()));
+                    String line;
+                    while ((line = reader.readLine()) != null) {
+                        output.append(line + "\n");
+                    }
+                    int exitVal = process.waitFor();
```

Comment:
We can remove the assignment since it is not used anywhere right? Or should it be used for some evaluation?
```suggestion
                    process.waitFor();
```

Reply:
Added exit code status